src/Cake.Common/Tools/ReportGenerator/Verbosity.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Represents ReportGenerator's logging verbosity.
+    /// </summary>
+    public enum Verbosity
```

Comment:
Yes. "Verbosity" will clash with `Cake.Core.Diagnostics.Verbosity` and should be renamed to `ReportGeneratorVerbosity`.